### PR TITLE
Updated cards

### PR DESCRIPTION
**Pull request by robhyrk**
- [x] update scalar markets (waiting for design)
- [x] confirm icons, categories, and. tags with design

### CODE REVIEW

File: components/markets/market-card/index.tsx [hunk 1]
```diff
@@ -45,26 +47,63 @@ const Pill = ({ value, classes }: { value: string; classes: string }) => {
...
+  isVerified,
+}: {
+  tags: string[];
+  baseAsset: string;
+  isVerified: boolean;
+}) => {
   return (
     <>
-      {tags?.map((tag, index) => {
+      {/* update later to include other assets */}
```
Comment (Robiquet) [inline components/markets/market-card/index.tsx:78]: Could you make this dynamic? Should be able to to if with `lookupAssetImagePath` or something similar

File: components/markets/market-card/index.tsx [hunk 2]
```diff
@@ -45,26 +47,63 @@ const Pill = ({ value, classes }: { value: string; classes: string }) => {
...
+              height={20}
+              src="icons/politics-cat-icon.svg"
+              alt="politics"
+            />
+          )
         );
-      })}
+      })} */}
+      {isVerified && (
+        <Image width={20} height={20} src="icons/verified-icon.svg" alt="" />
```
Comment (Robiquet) [inline components/markets/market-card/index.tsx:105]: Missing `alt` text

File: components/markets/MarketHeader.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ import { formatScalarOutcome } from "lib/util/format-scalar-outcome";
 import { Dialog } from "@headlessui/react";
 import { usePoolLiquidity } from "lib/hooks/queries/usePoolLiquidity";
 
-const UserIdentity: FC<
+export const UserIdentity: FC<
```
Comment (Robiquet) [inline components/markets/MarketHeader.tsx:30]: Could you move this to a separate file?

**Review by Robiquet — COMMENTED** [review]
Not too sure what's going on here? Prediction is 0 but the bar is in the middle
image

**Review by Robiquet — COMMENTED** [review]
Not too sure what's going on here? Prediction is 0 but the bar is in the middle
image

**Comment by Robiquet** [comment]
Could you remove the image fetching from IPFS? Seems like they need to be driven from the CMS

**Comment by lsaether** [comment]
Could we drop the word "Current" from the scalar bar so it just reads "Prediction:"?

**Comment by lsaether** [comment]
When you hover over the market creator you see the "+" interaction, but clicking on this still brings you to the market page. Do we need to define what the behavior should be here?

**Comment by robhyrk** [comment]
> When you hover over the market creator you see the "+" interaction, but clicking on this still brings you to the market page. Do we need to define what the behavior should be here?

This happens because the entire card is clickable to the market so clicking on anything within the card will cause that. I'm able to disable the avatar being clickable so I've done that in the latest update. 

The "+" that appears when you hover over it is supposed to copy the creator's address. I'm unsure if a user would find that useful on the cards, but we could explore keeping it or adding a different behaviour.  One way to make the avatar clickable is to make the top portion of cards un-clickable, thus making the creator avatar clickable.

**Comment by robhyrk** [comment]
Addressed issues mentioned and added scalar lower and upper bounds back discreetly. We could consider removing the bounds on the market cards and only keeping them on the market page as an alternative.

**Comment by Robiquet** [comment]
We might need to swap this image out as the background is showing up 
image

**Comment by lsaether** [comment]
@robhyrk Ok, I think that disabling the copy interaction on the avatar on the card makes sense!

**Review by Robiquet — APPROVED** [review]

**Pull request merged**